server/src/main/scala/services/CSVData.scala

```diff
@@ -198,16 +198,16 @@ object CSVData {
         fws.apiFlight.IATA,
         fws.apiFlight.ICAO,
         fws.apiFlight.Origin,
-        fws.apiFlight.Gate + "/" + fws.apiFlight.Stand,
+        fws.apiFlight.Gate.getOrElse("") + "/" + fws.apiFlight.Stand.getOrElse(""),
         fws.apiFlight.Status,
-        Try(SDate(fws.apiFlight.SchDT, europeLondonTimeZone).toISODateOnly).getOrElse(""),
-        Try(SDate(fws.apiFlight.SchDT, europeLondonTimeZone).toHoursAndMinutes()).getOrElse(""),
-        Try(SDate(fws.apiFlight.EstDT, europeLondonTimeZone).toHoursAndMinutes()).getOrElse(""),
-        Try(SDate(fws.apiFlight.ActDT, europeLondonTimeZone).toHoursAndMinutes()).getOrElse(""),
-        Try(SDate(fws.apiFlight.EstChoxDT, europeLondonTimeZone).toHoursAndMinutes()).getOrElse(""),
-        Try(SDate(fws.apiFlight.ActChoxDT, europeLondonTimeZone).toHoursAndMinutes()).getOrElse(""),
-        Try(SDate(fws.apiFlight.PcpTime, europeLondonTimeZone).toHoursAndMinutes()).getOrElse(""),
-        fws.apiFlight.ActPax,
+        Try(if (fws.apiFlight.Scheduled == 0) "" else SDate(fws.apiFlight.Scheduled, europeLondonTimeZone).toISODateOnly).getOrElse(""),
+        Try(if (fws.apiFlight.Scheduled == 0) "" else SDate(fws.apiFlight.Scheduled, europeLondonTimeZone).toHoursAndMinutes()).getOrElse(""),
+        Try(fws.apiFlight.Estimated.map(SDate(_, europeLondonTimeZone).toHoursAndMinutes()).getOrElse("")).getOrElse(""),
```

Comment:
we shouldn't need a Try on any of these anymore as we should have millis rather than a string.